src/app/clusters/closure-control-server/codegen_sources.gni

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2025 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+codegen_sources = [
+  "closure-control-cluster-delegate.h",
+  "closure-control-cluster-logic.cpp",
+  "closure-control-cluster-logic.h",
```

Comment:
Pretty sure the whole point here was that cluster logic and delegate (and likely cluster objects) do not depend on generated config.